doc/sphinx/source/conf.py

```diff
@@ -181,11 +181,10 @@
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    (master_doc, 'intelngraphlibrary', 'Intel nGraph Library',
+    (master_doc, 'ngraphcompiler', 'nGraph Compiler stack',
```

Comment:
is stack capitalized or not in the official name?

Reply:
Stack is not capitalized; Compiler must be when it's used alongside nGraph.

Reply:
ok thanks